buildscripts/builddrivers.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+#########################################################################################
+#
+# Description:  This script helps to build drivers in a Windows environment for PHP 7+ (32-bit/64-bit)
+#
+# Requirement:
+#               python 3.4
+#               PHP SDK and PHP Source 
+#               Driver source code folder / GitHub repository
+#               Visual Studio 2015 (PHP 7.0* and 7.1*) and Visual Studio 2017 (PHP 7.2*)
+#
+# Execution: Run with command line with required options.
+# Examples: 
+#           py builddrivers.py (for interactive mode)
+#           py builddrivers.py -v=7.1.7 -a=x86 -t=ts -d=all -g=no
+#           py builddrivers.py --PHPVER=7.0.22 --ARCH=x64 --THREAD=nts --DRIVER=all --GITHUB=yes
+#
+# Output: Build the drivers using PHP SDK. When running locally, if build is unsuccessful, 
+#         the log file will be launched for examination. Otherwise, the drivers will be renamed 
+#         and copied to the designated location(s).
+#
+#############################################################################################
+
+import sys
+import shutil
+import os.path
+import argparse
+from buildtools import BuildUtil
+
+class BuildDriver(object):
+    """Build sqlsrv and/or pdo_sqlsrv drivers with PHP source with the following properties:
+    
+    Attributes:
+        util            # BuildUtil object whose constructor takes phpver, driver, arch, thread, debug 
+        repo            # GitHub repository
+        branch          # GitHub repository branch
+        download_source # download source from GitHub or not
+        package         # package name for the binaries (will be ignored for local builds)
+        remote_path     # remote destination to where the drivers will be placed (None for local builds)
```

Comment:
My goal is to minimize the number of arguments, so `remote_path` is indeed *remote* because its default value **None** signifies this is a local build, which does things differently when it's not. For local build the expected behavior is to put the drivers in the 'ext' folder. So far there's no such requirement or use case to provide the flexibility to move the drivers anywhere else. Besides, the user (or php developer) can copy them to anywhere s/he likes afterwards. 

Since the user can run this script locally by providing arguments, if you insist I have to make this a generic *destination* path, I need some way to tell this is a local build. This means yet another argument, which seems to be something you're against based on your previous comments.